fortigate_exporter.go

```diff
@@ -54,6 +55,18 @@ type FortiHTTP interface {
 	Get(path string, query string, obj interface{}) error
 }
 
+func setUpMetricsEndpoint(version string, hash string) {
+	fortigateExporterInfo := promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "fortigate_exporter_info",
```

Comment:
fortigate_exporter_build_info maybe?
It seems like this might be a bit _too_ generic right now.

Reply:
Hmm I have looked into what other projects are using here some examples: 
```
grafana_build_info{branch="HEAD",edition="oss",goversion="go1.15.6",revision="c2203b9859",version="7.4.0"} 1
prometheus_build_info{branch="HEAD", goversion="go1.15.3", revision="0a7fdd3b76960808c3a91d92267c3d815c1bc354",  version="2.22.0"
```
I will change this to mimic other projects

Reply:
Regarding the application version. Should we strip the v from the tags, as Grafana and Prometheus do?     
So, basically `v1.1.1` vs `1.1.1` .

Reply:
Yes, let's strip the v if that is what the others do.